src/routes/subscriptions.rs

```diff
@@ -32,90 +33,31 @@ impl TryFrom<FormData> for NewSubscriber {
     }
 }
 
+#[derive(thiserror::Error)]
```

Comment:
`thiserror` has a macro that generates the boilerplate code we wrote by hand before